metafix/src/main/java/org/metafacture/metafix/Value.java

```diff
@@ -274,7 +270,7 @@ private void transformFields(final String[] fields, final UnaryOperator<String>
         private final Set<Type> expected = new HashSet<>();
         private final Value value;
 
-        private TypeMatcher(final Value value) {
+        /*package-private*/ TypeMatcher(final Value value) {
```

Comment:
Should be `private`.

Reply:
Not if you switch to `Value.matchType()`. See suggestions below.

Reply:
Right, saw it a moment after I commented (and deleted my comment).